CMP1903_A1_2324/Testing.cs

```diff
@@ -8,13 +9,19 @@ namespace CMP1903_A1_2324
 {
     internal class Testing
     {
-        /*
-         * This class should test the Game and the Die class.
-         * Create a Game object, call the methods and compare their output to expected output.
-         * Create a Die object and call its method.
-         * Use debug.assert() to make the comparisons and tests.
-         */
 
-        //Method
+        public void Tests()
+        {
+            Game game = new Game();
+            int sum = game.sumGame();
+            Debug.Assert(sum >= 3 && sum <= 18, "Sum of dice is not between 3 and 18.");
+            foreach (var die in game.dice)
+            {
+                Debug.Assert(die.dieRolled >= 1 && die.dieRolled <= 6, "Die is not between 1 and 6");
```

Comment:
Well done using Assert again to test for each die in your list to see if the rolls are as expected.